app/views/LoginView.tsx

```diff
@@ -44,6 +44,9 @@ const styles = StyleSheet.create({
 	},
 	loginButton: {
 		marginTop: 16
+	},
+	ugcContainer: {
```

Comment:
if bottomContainer style from file above have margin styles, why not move these style to the other file?